src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -56,6 +56,7 @@ const LaunchAnalysisButton = () => {
     const lastViewed = moment().toISOString();
     dispatch(updateExperiment(experimentId, { lastViewed }));
     dispatch(updateProject(activeProjectUuid, { lastAnalyzed: lastViewed }));
+    dispatch(switchExperiment());
```

Comment:
we were talking that you will call this from the content wrapper. Why only here?

Reply:
As discussed with Agi yesterday, because if you open the same experiment again in the browser it will remove everything. This way we make sure we clean the store only when launch analysis is pressed.

Reply:
Experiment pollution could only happen when going back from a page with experiment, to data management, and then choosing another experiment/project. So we should reset the store when moving from a page with out experiment id to a page with experiment id. That's why it's set at the launch button.

Changing the experimentId directly from the URL should cause a hard reset, which will cause the redux store to be recreated. A hard reset will cause the redux store to reset right?

Am I understanding correctly or did I miss something?

Reply:
Yes hard reset would cause it correct